### PR TITLE
feat(index): update style to sciwork theme

### PR DESCRIPTION
**Pull request by rockleona**
## Pull request type

Please check the type of change your PR introduces:

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe):

## What is the current behavior?

Previous landing page (index.html) had COSCUP style and words.

## What is the new behavior?

- Update landing page with sciwork style

### CODE REVIEW

**Comment by rockleona** [comment]
Here is the preview of the change:

<details>
<summary>Screenshot</summary>

image

</details>

**Review by chestercheng — CHANGES_REQUESTED** [review]
* I have created the dev branch as the default branch, so please change the base to it.
* I think we can temporarily remove the “About Volunteer Groups” and “Calendar” sections on the index page.
* The “About sciwork” section still mentions COSCUP. Please correct it and use the content from our portal instead.

**Comment by rockleona** [comment]
> * I have created the dev branch as the default branch, so please change the base to it.
> * I think we can temporarily remove the “About Volunteer Groups” and “Calendar” sections on the index page.
> * The “About sciwork” section still mentions COSCUP. Please correct it and use the content from our portal instead.

Hi @chestercheng, I already push a commit with your suggestion, please review

File: templates/index.html [hunk 1]
```diff
@@ -153,77 +153,26 @@
...
-                        <li>是舉辦與 <strong>Open Source</strong> 相關議題的科技研討會。</li>
-                        <li>參與對象是開發者、使用者、推廣者（Coders, Users, Promoters）。</li>
-                        <li>是一個<strong>不需要付費</strong>、也<strong>不需要報名</strong>的研討會。</li>
-                        <li>通常會在七月底、八月初在台科大舉辦活動。</li>
-                        <li>每一議程軌都是由各<strong>開源社群夥伴</strong>自審、自辦。</li>
-                        <li>大約會有 1500 - 2000 人來參與的研討會。</li>
-                    </ul>
-                    <p>登錄成為 COSCUP 志工，每屆活動開始時，就有機會一起參與籌備。來這裡發揮你的專長與熱情，一起籌辦年度大型研討會活動，來這裡實驗、實現你的想法與規劃！</p>
                     <div class="has-text-right">
                         <a href="https://coscup.org/"
```
Comment (chestercheng) [inline templates/index.html]: there is a typo, it should be https://sciwork.dev

**Review by chestercheng — CHANGES_REQUESTED** [review]

**Review by chestercheng — APPROVED** [review]
LGTM

**Comment by chestercheng** [comment]
Since we will redesign the UI, I think this PR can be closed

**Pull request closed without merging**